src/clients/remote.rb

```diff
@@ -56,8 +56,6 @@ def main
         ),
         "id"         => "remote",
         "guihandler" => fun_ref(method(:RemoteGUI), "any ()"),
-        "initialize" => fun_ref(Remote.method(:Read), "boolean ()"),
-        "finish"     => fun_ref(Remote.method(:Write), "boolean ()"),
         "actions"    => {
           "list"  => {
```

Comment:
You're right... `list` cannot know the status of the system without initialization.
I'll fix this.